department-of-veterans-affairs/va.gov-cms
# Add help, label, and edit form changes to config.

## Description
Contains help text edits etc from onhelptext demo environment

See #1828

## QA steps
- Check that help text, labels, and edit forms appear as expected.
- Double-check that only appropriate config files have been updated.
